### ISSUE TITLE
All math types affected: `.ToString()` throws `System.NullReferenceException`

### ISSUE THREAD

**Issue by BoyBaykiller** [opentk/opentk#1572]
### Description

Due to recent changes also included in OpenTK 4.7.6, calling the `ToString()` method for math types like `Vector3` or `Matrix4` results in a `System.NullReferenceException` exception.

### Repro steps

```cs
Vector3 vec = new Vector3(1.0f); // or some other math type

// throws System.NullReferenceException
vec.ToString();

// also throws System.NullReferenceException
Console.WriteLine($"{vec}");
```
### Related information 

* Guessing the issue was introduced in one of those commits:
  - f4d6170b6e5b22fd6548c83fc1b86295ffa26058
  - c2dc92b5491e954a6b814b8d5e0d8135e24c739f
* OpenTK.Mathematics 4.7.6

**Comment by NogginBops** [comment]
Lord, should have tested that PR a bit more...
Will push a fix and make a new release asap

**Comment by NogginBops** [comment]
This has been fixed with c485e521fdfc2334e1a2d18e6538a0bc2a9b8bcc and 4.7.7 has been released.
